types/errors/link.go

```diff
@@ -0,0 +1,12 @@
+package errors
```

Comment:
How about changing the file name to `legacy_error.go` or others, not `link.go`?

Reply:
These errors would be a spec, and last as long as x/token and x/collection would. So I think it should be fine as is (I don't like these errors though).

Reply:
I added a comment above these errors.